core/src/main/java/org/infinispan/distribution/ch/ConsistentHashFactory.java

```diff
@@ -104,11 +84,19 @@ default CH create(Hash hashFunction, int numOwners, int numSegments, List<Addres
    CH union(CH ch1, CH ch2);
 
    /**
-    * Recreates a ConsistentHash from a previously stored persistent state. The returned ConsistentHash will not have
-    * proper addresses, but {@link org.infinispan.topology.PersistentUUID}s instead so they will need to be replaced
-    * @param state the state to restore
+    * Recreates a {@link ConsistentHash} from a previously stored persistent state.
+    * <p>
+    * The stored state typically contains a collection of {@link PersistentUUID}s representing the members. If a member
+    * is not present when reading from the state, its {@link PersistentUUID} must be added to the
+    * {@link PersistedConsistentHash#missingUuids()} list. The {@link PersistedConsistentHash#consistentHash()} may have
+    *  incomplete ownership when there are missing UUIDs.
+    *
+    * @param state         the state to restore, containing member {@link PersistentUUID}s.
+    * @param addressMapper A function to map the {@link PersistentUUID} to {@link Address}.
+    * @return A {@link PersistedConsistentHash} with the {@link ConsistentHash} and with the missing
+    * {@link PersistentUUID} if any.
     */
-   default CH fromPersistentState(ScopedPersistentState state) {
```

Comment:
Same question as the ConstitentHash methods?

Reply:
Answering both questions, we say `@deprecated Since 11.0. Will be removed in 14.0, the segment allocation will no longer be customizable.`
It is enough for me to remove the methods but I don't mind adding them back. The old methods are no longer invoked and I don't see a reason for the user to invoke them in their code. Let me know what you think.

Reply:
That's fine by me. I missed the fact that we have deprecated `ConsistentHashFactory`. I had a quick look at removing this myself, but it affects a *LOT* of tests.